### DISCUSSION TITLE
Expand_as RuntimeError in torch_scatter!!!

### DISCUSSION THREAD

**Discussion by tjb-tech** [pyg-team/pytorch_geometric#5719] [Q&A]
Hi, I am inheritanting Messagepassing in pytorch_geometric to create my own graph conv, but I face the error below. To be exact, it occurs in function `propagate`, and I sent edge_index (2, edge_num) and x (num_node, 128) into propagate. And I try my code on both linux and win. And they occurred the same problems. Note that I use the famous dataset `Cora` use the code `dataset = Planetoid(path, 'Cora', transform=T.NormalizeFeatures())`
linux: 
![J4U6NM5}) )(0RA2LZBQG8K](https://user-images.githubusercontent.com/56426168/195987998-b3eb6b03-1fdb-4d75-942b-94ed3625a2dc.png)
Win: 
![L9~ GBZ1WKSA{{0S1W M08Q](https://user-images.githubusercontent.com/56426168/195988011-f98071b0-ab6b-40fd-bf09-8f03819a8eaa.png)
BTW, linux is py3.8 and Win is py3.7.

**Comment by rusty1s** [comment]
Thanks for reporting. It looks like your node feature matrix is three-dimensional. In that case, you should set `super().__init__(aggr=.., node_dim=0)` in your `MessagePassing` constructor.

**Reply by tjb-tech** [reply]
Thanks a lot for replying!
